### DISCUSSION TITLE
Vue 2 JS

### DISCUSSION THREAD

**Discussion by Amanda19951231** [aws/aws-iot-device-sdk-js-v2#473] [General]
Hello people!
I have a question for you guys.
Have somebody implemented this library in a project with vue 2?

**Comment by jmklix** [comment]
We have not implemented/tested this sdk in a vue 2 project and since vue 2 is past end of life (EOL was December 31st, 2023) this probably won't be supported. But anyone else is free to share there projects.

**Reply by prabh-netomi** [reply]
Yes, we implemented this library in Vue 2 with webpack5. There’s no direct dependency on the Vue framework — we created a separate MQTT class and simply call its methods from the component code.
